packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -534,11 +541,27 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
         return nodes;
     }
 
-    protected filenameAndPath(rootUriStr: string, uriStr: string): { name: string, path: string } {
+    /**
+     * Get file name and construct a path of a file relatively to its root.
+     * @param rootUriStr the string of root uri.
+     * @param uriStr the string of the file uri.
+     * @param getAbsolutePath option to get absolute path instead of relative path.
+     */
+    protected filenameAndPath(rootUriStr: string, uriStr: string, getAbsolutePath?: boolean): {
```

Comment:
```suggestion
    protected filenameAndPath(rootUriStr: string, uriStr: string, absolute?: boolean): {
```

I believe this name would be sufficient.